packages/terra-functional-testing/tests/wdio/terra-validates-spec.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * This spec file is used for integration testing the Terra.validates commands.
+ */
+
+describe('Axe', () => {
```

Comment:
```suggestion
describe('Terra.validates', () => {
```

Reply:
Updated to `Terra.validates.accessibility`:

https://github.com/cerner/terra-toolkit/pull/452/commits/1aad2e2c77dc3a59fed32b3ce8c763a95b57d952